### PR TITLE
[expo] Send device platform back to CLI

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

- https://github.com/expo/expo-cli/pull/3769

# Test Plan

- Added this code to the local node module of expo and used the https://github.com/expo/expo-cli/pull/3769 branch of expo-cli

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by ide — APPROVED** [review]

File: packages/expo/src/logs/RemoteLogging.ts [hunk 1]
```diff
@@ -112,6 +113,7 @@ async function _sendNextLogBatchAsync(batch: LogEntry[], logUrl: string): Promis
     Accept: 'application/json',
     'Device-Id': await getInstallationIdAsync(),
     'Session-Id': _sessionId,
+    'Device-Platform': Platform.OS,
```
Comment (EvanBacon) [inline packages/expo/src/logs/RemoteLogging.ts:116]: Actually noticing now that we use `expo-platform` in a few places as a query param: https://github.com/expo/expo-cli/blob/4a1d73ac49db75233a3372028f78f41df783fbee/packages/xdl/src/start/ExpoUpdatesManifestHandler.ts#L25 would it maybe make sense to use that param just for parity?

Comment (ide) [reply packages/expo/src/logs/RemoteLogging.ts:116]: If we think the purpose of the existing expo-platform header and the one you're adding here are going to be the same indefinitely, I would use the same header. If the headers happen to share the same values right now but their purposes may diverge in the future, I would use a different header.

File: packages/expo/src/logs/RemoteLogging.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import { Platform } from '@unimodules/core';
```
Comment (tsapeta) [inline packages/expo/src/logs/RemoteLogging.ts:1]: We should already import from `expo-modules-core` 😉 but you can leave it as is, I'm going to rename all existing imports soon

**Review by tsapeta — APPROVED** [review]

File: packages/expo/src/logs/RemoteLogging.ts (hunk 2 above)
Comment (tsapeta) [reply packages/expo/src/logs/RemoteLogging.ts:1]: Ok, let's fix it here as I've already renamed existing imports
```suggestion
import { Platform } from 'expo-modules-core';
```

**Pull request merged**